tensorflow/training/docker/2.0.1/py3/Dockerfile.cpu

```diff
@@ -25,6 +25,9 @@ ARG PYTHON_PIP=python3-pip
 ARG PIP=pip3
 
 ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.1/AmazonLinux/cpu/final/tensorflow-2.0.1-cp36-cp36m-manylinux2010_x86_64.whl
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=0.8.1
```

Comment:
there is no pip install smdebug in this file. so this arg wont be used. same with the gpu Dockerfile

Reply:
Added in the next commit